packages/usa-date-picker/src/index.js

```diff
@@ -721,6 +721,7 @@ const disable = (el) => {
 const ariaDisable = (el) => {
   const { externalInputEl, toggleBtnEl } = getDatePickerContext(el);
 
+  el.setAttribute("readonly", "")
```

Comment:
**issue (blocking):** This is incorrectly placing the `readonly` attribute on the `usa-date-picker` div element instead of the input element.

Can you update this to append the attribute to the correct element and confirm that the text box is preventing input?

Reply:
Resolved in [14f5cb1](https://github.com/uswds/uswds/pull/6013/commits/14f5cb112eaa1dec968244878608f653557cd734)